bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -244,6 +244,15 @@ type NodeRegistrationOptions struct {
 	// IgnorePreflightErrors provides a slice of pre-flight errors to be ignored when the current node is registered.
 	// +optional
 	IgnorePreflightErrors []string `json:"ignorePreflightErrors,omitempty"`
+
+	// ImagePullPolicy specifies the policy for image pulling
+	// during kubeadm "init" and "join" operations. The value of
+	// this field must be one of "Always", "IfNotPresent" or
+	// "Never". Defaults to "IfNotPresent".
+	// +kubebuilder:validation:Enum=Always;IfNotPresent;Never
+	// +kubebuilder:default=IfNotPresent
+	// +optional
+	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
```

Comment:
q: should we add a note explaining that this feature can be used only with Kubernetes >=1.22 (please check version)?

Reply:
Good catch. We should have similar comments on some of our other fields IIRC.

Reply:
Fixed.
Verified Kubernetes version in which imagePullPolicy is added. https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md

Reply:
Thank you!